ui/public/locales/zh/translation.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": {
```

Comment:
I wonder if we should just leave these all uncategorized as a single list. Since things like `entrypoint` and `parameters` are not necessarily titles and that can get very semantic very fast.

at the very least, I do think every entry should be alphabetized

Reply:
Sure.